app/models/question.rb

```diff
@@ -22,6 +22,27 @@ def self.to_csv
     end
   end
 
+  # Structure of the file should match the self.to_csv
+  # content of question -> content of correct answer -> array of content of
+  # incorrect answers
+  def self.import file, user
+    CSV.foreach( file.path, col_sep: ',', headers: true ) do |row|
+      correct_ans = Answer.create!(content: row[1], correct: true)
+      incorrect_ans = []
+      row[2].tr('[','').tr(']','').split(',').each do |r|
```

Comment:
Hacky, I know. The exported csv has all incorrect answers as an array that's been stringified. This is necessary to remove the brackets at the beginning and end, then splits the string on the comma.